src/test/unit/test_lumigo_utils.py

```diff
@@ -411,22 +415,70 @@ def test_report_json_china_missing_secret_access_key(monkeypatch, reporter_mock,
     )
 
 
-def test_report_json_china_no_boto(monkeypatch, reporter_mock, caplog):
+@pytest.mark.parametrize("package_name", ["boto3", "botocore"])
+def test_report_json_china_no_boto(package_name, monkeypatch, reporter_mock, caplog):
     reporter_mock.side_effect = report_json
     monkeypatch.setattr(Configuration, "should_report", True)
     monkeypatch.setattr(Configuration, "edge_kinesis_aws_access_key_id", "my_value")
     monkeypatch.setattr(Configuration, "edge_kinesis_aws_secret_access_key", "my_value")
-    monkeypatch.setattr(lumigo_utils, "boto3", None)
```

Comment:
This is a bit non-realistic test - botocore is a dependency to boto3, thus it is not possible that boto3 will be installed and botocore not 🤷